src/authz-module/index.scss

```diff
@@ -8,3 +8,11 @@
     background-color: var(--pgn-color-light-200);
   }
 }
+
+.toast-container {
+  // Ensure toast appears above modal
+  z-index: 1000;
+  // Move toast to the right 
+  left: auto;
+  right: var(--pgn-spacing-toast-container-gutter-lg);
+}
```

Comment:
I know we've run into issues when overriding toast container styles in the past (see https://github.com/openedx/frontend-app-authoring/issues/1898)

I haven't done exhaustive testing of this, but I was able to replicate some of the strange behavior I saw in the authoring issue

https://github.com/user-attachments/assets/c265154f-d54d-4519-9fab-229a44759f9f